opennms-doc/guide-admin/src/asciidoc/text/poller/VmwareCimMonitor.adoc

```diff
@@ -0,0 +1,60 @@
+
+=== VmwareCimMonitor
+
+This monitor is part of the VMware integration provided in _Provisiond_.
+The monitor is specialized to test the health status provided from all _Host System_ (host) sensor data.
+
+IMPORTANT: This monitor is only executed if the host is in power state _on_.
+
+IMPORTANT: This monitor requires to import hosts with _Provisiond_ and the _VMware_ import.
+           OpenNMS requires network access to _VMware vCenter_ and the hosts.
+           To get the sensor data the credentials from 'vmware-config.xml' for the responsible _vCenter_ is used.
+           The following asset fields are filled from _Provisiond_ and is provided by _VMware_ import feature: 
+           _VMware Management Server_, _VMware Managed Entity Type_ and the _foreignId_ which contains an internal _VMware vCenter Identifier_.
+
+The global health status is evaluated by testing all available host sensors and evaluating the state of each sensor.
+A sensor state could be represented as the following:
+
+ - _Unknown(0)_
+ - _OK(5)_
+ - _Degraded/Warning(10)_
+ - _Minor failure(15)_
+ - _Major failure(20)_
+ - _Critical failure(25)_
+ - _Non-recoverable error(30)_
+
```

Comment:
@christianpape Regarding the enhancement of the `ignoreStandBy` this seems not be correct. Is there something missing? The current documentation explains _VmwareMonitor_ test if *VM* or *Host* is powered up. The VmwareCimMonitor just monitors the sensor state. What does it mean to have here the _ignoreStandBy_ parameter?